clients/drcachesim/drcachesim.dox.in

```diff
@@ -421,6 +422,89 @@ dcache top 10
     0x7ffcc35e7e40: 1997
 \endcode
 
+****************************************************************************
+\section sec_drcachesim_config_file Configuration File
+
+\p drcachesim supports reconfigurable cache hierarchies defined in
+a configuration file. The configuration file is a text file with the following
+formatting rules.
+
+- A comment starts with two slashes followed by one or more spaces. Anything
+after the '// ' till the end of the line is considered a comment and ignored.
+- A parameter's name and its value are listed consecutively with white space
+(spaces, tabs, or a new line) between them.
+- Parameters must be separated by white space. Including one parameter per line
+helps keep the configuration file more human-readable.
+- A cache's parameters must be enclosed inside braces and preceded by the
+cache's unique name.
+- Parameters can be listed in any order.
+- Parameters not included in the configuration file take their default values.
+- String values must not be enclosed in quotations.
+
+Supported common parameters and their values types:
+- num_cores <unsigned int>
+- line_size <unsigned int>
+- skip_refs <unsigned int>
+- warmup_refs <unsigned int>
+- warmup_fraction <float in [0,1]>
+- sim_refs <unsigned int>
+- cpu_scheduling <bool>
+- verbose <unsigned int>
+
+Supported cache parameters and their values types:
+- type <string, one of instruction, data, or unified>
+- core <unsigned int in [0, num_cores)>
+- size <unsigned int, power of 2>
+- assoc <unsigned int, power of 2>
+- inclusive <bool>
+- parent <string>
+- replace_policy <string, one of >
+- prefetcher <string>
+- miss_file <string>
+
+Example:
+// Configuration for a single-core CPU.
+
+// Common params.
+num_cores       1
+line_size       64
+cpu_scheduling  true
+sim_refs        8888888
+warmup_fraction 0.8
+
+// Cache params.
+P0L1I {                        // P0 L1 I$
+  type          instruction
+  core          0
+  size          65536
```

Comment:
suggest: 64K, here and below

Reply:
This is intentional to demonstrate that both formats (i.e. 65536 and 64K) are supported. I used the other format for the L2 and LLC caches.